src/main/java/org/littleshoot/proxy/extras/SelfSignedSslEngineSource.java

```diff
@@ -150,8 +234,7 @@ public void checkServerTrusted(X509Certificate[] arg0,
             sslContext = SSLContext.getInstance(PROTOCOL);
             sslContext.init(keyManagers, trustManagers, null);
         } catch (final Exception e) {
-            throw new Error(
-                    "Failed to initialize the server-side SSLContext", e);
+            throw new Error("Failed to initialize the server-side SSLContext", e);
```

Comment:
What 's more, en error means that serious problems have occured,  and a reasonable application should not try to catch it.